edgelet/contrib/debian/prerm

```diff
@@ -2,7 +2,13 @@
 set -e
 
 case "$1" in
-    remove|upgrade|deconfigure)
+    remove|deconfigure)
+        if [ -f /etc/iotedge/config.yaml ]; then
+            rm /etc/iotedge/config.yaml
+        fi
+    ;;
```

Comment:
It looks like we're changing the behavior here to delete the user's `config.yaml` on uninstall...@arsing is that what we want?

Reply:
It ought to be only on `purge`, not on `remove` (ref https://github.com/Azure/iotedge/pull/2301#issuecomment-573898598 )

Reply:
@gri6507 Can you confirm that the `rm` only runs on `purge`, not on `remove` ?

Reply:
@arsing  `prerm` does not get called for `purge`. Therefore, it has to be on `remove`. (see https://www.debian.org/doc/debian-policy/ap-flowcharts.html for more info). The problem with moving the deletion to `postrm purge` is that you loose the ability to reconfigure the package because `reconfigure` is not an option for `postrm`. 

Since earlier in this PullRequest we were considering using `DebConf`, the `reconfigure` is critical. However, now that we are no longer using DebConf, I will move back to deleting in `postrm purge` .